infrastructure/code/code.go

```diff
@@ -207,77 +192,18 @@ func (sc *Scanner) waitForScanToFinish(scanStatus *ScanStatus, folderPath string
 	return false
 }
 
-func (sc *Scanner) files(folderPath string) (filePaths []string, err error) {
-	t := progress.NewTracker(false)
-	workspace, err := filepath.Abs(folderPath)
-	t.Begin(fmt.Sprintf("Snyk Code: Enumerating files in %s", folderPath), "Evaluating ignores and counting files...")
-
-	if err != nil {
-		return filePaths, err
-	}
-	sc.mutex.Lock()
-	var fileCount int
-	var ignorePatterns []string
-	ignorePatterns, fileCount, err = sc.loadIgnorePatternsAndCountFiles(folderPath)
-	if err != nil {
-		return filePaths, err
-	}
-
-	gitIgnore := ignore.CompileIgnoreLines(ignorePatterns...)
-	sc.mutex.Unlock()
-	filesWalked := 0
-	log.Debug().Str("method", "folder.Files").Msgf("File count: %d", fileCount)
-	err = filepath.WalkDir(
-		workspace, func(path string, dirEntry os.DirEntry, err error) error {
-			if err != nil {
-				log.Debug().
-					Str("method", "domain.ide.workspace.Folder.Files").
-					Str("path", path).
-					Err(err).
-					Msg("error traversing files")
-				return nil
-			}
-			filesWalked++
-			percentage := math.Round(float64(filesWalked) / float64(fileCount) * 100)
-			t.ReportWithMessage(
```

Comment:
this seems to be missing in the new implementation (but is important for user progress reporting on big repos).